Transplant blurb on the choice of the name "Official Images" (#19)
docker-library/faq: README.md

### What do you mean by "Official"?

> The name of this program was chosen in an attempt to reflect that upstream-first focus (although in hindsight, it's clear that it was a choice with some amount of confusion potential for which we're sorry). 

Reviewer: ```suggestion
The name of this program was chosen in an attempt to reflect our upstream-first focus (although in hindsight, it's clear that it was a choice with some amount of confusion potential for which we're sorry). 
```

> This question is so common it's answered in our primary repository! See [the answer in the readme of the `github.com/docker-library/official-images` repository](https://github.com/docker-library/official-images#what-do-you-mean-by-official).

Reviewer: Given we're ditching this section in that readme, we should probably reword this a bit:

```suggestion
See [the readme of the `github.com/docker-library/official-images` repository](https://github.com/docker-library/official-images#what-are-official-images) for a more verbose overview of the program.
```

> This question is so common it's answered in our primary repository! See [the answer in the readme of the `github.com/docker-library/official-images` repository](https://github.com/docker-library/official-images#what-do-you-mean-by-official).

Reviewer 2: The link to "what-do-you-mean-by-official" will no longer work after https://github.com/docker-library/official-images/pull/9379.
